config/config.yml

```diff
@@ -460,6 +460,7 @@ PERMITTED_HOSTS: [
   "archiveofourown.org",
   "download.archiveofourown.org",
   "insecure.archiveofourown.org",
+  "secure.archiveofourown.org",
```

Comment:
Can we add the IP address mentioned in the issue too? Or is there a reason we shouldn't do that?

Reply:
I am moving over to a new ip address before we balance over a new pair, I don't really support the bare ip address although we could for http but not https.

https://208.85.241.152/ is a security warning as expected. I could make http://208.85.241.152/  redirect to https:archiveofourown.org/ I think.

If I did that we still wouldn't want the ip address in it ?

Reply:
Support says people use the IP address to get around DNS poisoning. Would redirecting affect their ability to do that?

Reply:
I will add the 2 ip's then.

Reply:
fwiw, I'm getting 503s on the IP addresses (using HTTP), but I assume you're aware of that